flytekit/lazy_import/lazy_module.py

```diff
@@ -24,6 +34,10 @@ def lazy_module(fullname):
         return sys.modules[fullname]
     # https://docs.python.org/3/library/importlib.html#implementing-lazy-imports
     spec = importlib.util.find_spec(fullname)
+    if spec is None:
+        # Return a dummy module if the module is not found in the python environment,
+        # so that we can raise a proper error when the user tries to access an attribute in the module.
+        return DummyModule(fullname)
```

Comment:
Is `tests/flytekit/unit/lazy_module/test_lazy_module.py` not being run by Codecov? I'm pretty sure your function `test_lazy_module` should cover all the new lines in this file.

Reply:
Codecov always has this issue, we haven't figure it out yet